src/tribler/core/utilities/tiny_tribler_service.py

```diff
@@ -51,7 +53,7 @@ async def start_tribler():
         if fragile:
             make_async_loop_fragile(loop)
 
-        loop.create_task(start_tribler())
+        _ = loop.create_task(start_tribler())
```

Comment:
The code looks correct, but for future developers, the importance of this "underscore" variable might not be clear, and at some moment, it may be removed as part of some refactoring.

For this reason, it may be better to have a named variable. As IDE and linters can complain about unused variable, it is possible to use a new attribute instead:
```python
self.main_task = loop.create_task(start_tribler())
```
(I think it is also possible to use `self.async_group.add` here instead of `loop.create_task`)

Reply:
Fixed